spec/features/granules/granule_filters_spec.rb

```diff
@@ -18,7 +19,7 @@
 
     before_granule_count = 0
     synchronize do
-      number_granules = expect(page.text).to match /\d+ Granules/
+      number_granules = expect(find('#project-collections-list').text).to match /\d+ Granules/
```

Comment:
Use helpers defined in overlay_helpers.rb or define new ones as needed. I think `expect(project_overview).to ...` will do what you want here.